modules/ossm-accessing-jaeger.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/-ossm-troubleshooting-istio.adoc
+
+[id="ossm-accessing-jaeger_{context}"]
+= Accessing the Jaeger console
+////
+(how to find the URL)
+Installed Operators > Jaeger Operator > Jaeger > Jaeger Details > Resources > Route > Location = Link
+Networking > Routes> search Jaeger route (Location = Link)
+Kiali Console > Distributed Tracing tab
+////
+
+The installation process creates a route to access the Jaeger console.
+
+. Log in to the {Product-title} console.
+
+. Navigate to *Networking* -> *Routes* and
+search for the Jaeger route, which is the URL listed under Location.
+
+. To query for details of the route using the command line, enter the following command.  In this example, `istio-system` is the control plane namespace.
++
+[source,terminal]
+----
+$ export JAEGER_URL=$(oc get route -n istio-system jaeger -o jsonpath='{.spec.host}')
+----
++
+. Launch a browser and navigate to \https://<JAEGER_URL>, where <JAEGER_URL> is the route that you discovered in the previous step.
```

Comment:
Pretty sure the export command is silent, just runs and stores it in the variable without printing anything to the screen... would it be better to either use the variable or just do the oc get directly without `export JAEGER_URL`?